init-tools.sh

```diff
@@ -37,7 +37,14 @@ if [ -z "$__DOTNET_PKG" ]; then
             __DOTNET_PKG=dotnet-dev-linux-x64
             OS=Linux
 			
-            if [ -e /etc/redhat-release ]; then
+            if [ -e /etc/os-release ]; then
+                source /etc/os-release
+                if [[ $ID == "alpine" ]]; then
+                    # remove the last version digit
+                    VERSION_ID=${VERSION_ID%.*}
```

Comment:
is there a reason to strip the last digit of the version? Why fight the system here and just let the RID include that.

Reply:
The last number component of the version isn't important, it is just an update version. It is like the second version component on RHEL. It doesn't seem to make sense to treat each minor update as a separate rid and have a separate tarball for it.